app/src/lib/git/reflog.ts

```diff
@@ -63,3 +63,41 @@ export async function getRecentBranches(
 
   return [...names]
 }
+
+/**
+ * Gets the distinct list of branches that have been checked out after a specific date
+ */
+export async function getCheckoutsAfterDate(
+  repository: Repository,
+  afterDate: Date
+): Promise<ReadonlyArray<string>> {
+  //regexr.com/46n1v
```

Comment:
This is a neat way to reference the regex test suite, but I'd also like to see some tests against a real Git repository so we can be sure we're getting the right values from the repository.